scripts/reset-localhost-projects.py

```diff
@@ -0,0 +1,114 @@
+import os
+import argparse
+from getpass import getpass
+
+import requests
+
+
+def main():
+    user = CommandLineParser().parse()
+    API(user).parse()
+
+
+class API:
+    def __init__(self, user):
+        self.target = "http://localhost:8080/"
+        self.user = user
+        self.header = self._create_header()
+
+    def parse(self):
+        projects = self._parse_projects()
+        for project in projects:
+            if project != "lifecycle":
+                self._delete_project(project)
+        users = self._parse_users()
+        for user in users:
+            if user != self.user:
+                self._delete_user(user)
+
+    def _parse_projects(self):
+        projects = []
+        localhost_project_dict = self._get_projects()
+        for project in localhost_project_dict:
+            projects.append(project["name"])
+        return projects
+
+    def _parse_users(self):
+        users = []
+        localhost_user_dict = self._get_users()
+        for user in localhost_user_dict:
+            users.append(user["email"])
+        return users
+
+    @staticmethod
+    def _create_header():
+        token = getpass("Token: ")
+        return {"Authorization": "Bearer " + token}
+
+    def _delete_project(self, project_name):
+        response = requests.delete(f"{self.target}access/projects/{project_name}", headers=self.header)
+        response.raise_for_status()
+
+    def _delete_user(self, user_email):
+        response = requests.delete(f"{self.target}access/users/{user_email}", headers=self.header)
+        response.raise_for_status()
+
+    def _get_projects(self):
+        response = requests.get(f"{self.target}access/projects", headers=self.header).json()
+        return response
+
+    def _get_users(self):
+        response = requests.get(f"{self.target}access/users", headers=self.header).json()
+        return response
+
+
+class CommandLineParser:
+    def __init__(self):
+        self.parser = CommandLineInterface()
+
+    def parse(self):
+        su = self._parse_su()
+        return su
+
+    def _parse_su(self):
+        return self.parser.get_argument("user")
```

Comment:
- What means `su`? Maybe expand in full word(s)?
- `_parse_su` is private and only called from `parse`? ... then you could fold code into `parse` itself no?

Reply:
Hmm ... `su` is super user .. or is this users with admin role? (I myself thing linux `su`)

Reply:
It was meant as superuser yes, changed for clarification